tests/detect-vlan-id/test.yaml

```diff
@@ -0,0 +1,22 @@
+requires:
+ min-version: 8
+
+args:
+- -k none
+
+checks:
+- filter:
+   count: 1
+   match:
+     event_type: alert
+     alert.signature_id: 1
```

Comment:
Could we check the vlan id in the alert data ?

Reply:
Sorry, I didn't understand the question. Could you be more specific, please?

Reply:
Can we we add in this match section a check about `vlan.id: 200` ? (with the good syntax and value)

Reply:
I don't think so. Should I use these other commits as examples for the implementation?
- https://github.com/OISF/suricata/pull/11945/commits/a8fe7a54e70c4d19c4c2a0accd0e38996f023341
- https://github.com/OISF/suricata-verify/pull/2088/commits/e5861a895ceb07b6af667052156a923309983132

Reply:
schema.json tells me that it should be `vlan[0]: 200`